portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/site/PortalSiteHelperImpl.java

```diff
@@ -459,8 +459,9 @@ public Map convertSiteToMap(HttpServletRequest req, Site s, String prefix,
 				&& (s.getId().equals(myWorkspaceSiteId) || effectiveSite
 						.equals(myWorkspaceSiteId))));
 		
+		String siteTitleRaw = getUserSpecificSiteTitle(s, false, false, siteProviders);
 		String siteTitle = Validator.escapeHtml(getUserSpecificSiteTitle(s, false, false, siteProviders));
```

Comment:
I would suggest the following, to avoid 2 calls to `getUserSpecificSiteTitle()`:

```suggestion
		String siteTitle = Validator.escapeHtml(siteTitleRaw);
```

Reply:
Precisely!